src/regions/ca/info/faq/39_gloves.md

```diff
@@ -0,0 +1,13 @@
+## Do I have to wear latex or vinyl gloves when I go to the supermarket?
+
+Just as wearing a mask, wearing gloves also offers a false sense of security, as well as a risk of contamination during improper handling.
+
+It is reserved for use by health or children care professionals when there is an immediate risk of biological fluids exposure.
+
+Wearing gloves is not a substitute for hand hygiene, and if used, gloves should be used for the shortest possible time.
+
+The integrity of gloves is sensitive to the use of products, time, moisture in your hands. If you wear gloves because you are caring for someone who is infected, you should wash your hands before and after use.
+
+This is how to care for a person with covid-19 : https://www.canada.ca/en/public-health/services/publications/diseases-conditions/how-to-care-for-person-with-covid-19-at-home-advice-for-caregivers.html
```

Comment:
Can we change this so it is not shown as a link in the page?

Reply:
fixed!